src/components/Create/components/RewardsList/RewardItem.tsx

```diff
@@ -66,7 +73,14 @@ export const RewardItem = ({
 
       <div className="flex flex-col gap-8 md:flex-row">
         <div className="flex flex-col gap-3">
-          <RewardImage className="h-44 w-44" src={fileUrl.toString()} />
+          {isVideo ? (
+            <div className="relative h-44 w-44">
+              <JuiceVideoThumbnail src={fileUrl.toString()} />
+            </div>
+          ) : (
+            <RewardImage className="h-44 w-44" src={fileUrl.toString()} />
+          )}
```

Comment:
**suggestion** this is starting to feel like tech debt. I recall theres at least 2 other spots where we do this `isVideo` switch. Perhaps at a minimum we should have components for each of these: `NftRewardMedia.tsx` (and whatever other ones are necessary). WDYT?

Reply:
Yeah I feel that. There are 4 instance of `isVideo`. Although I think that at least 3 if not all a unique and creating a single component will take a bit of messing around. 

- one is `antd image`/`JuiceVideoThumbnail` (thumbnail shows no video controls and some other differences
- one is `img` / `JuiceVideoPreview` (preview shows video controls)
- `img` / `JuiceVideoThumbnail`
- `JuiceVideoThumbnail` / *this*:
<img width="688" alt="Screen Shot 2023-02-10 at 12 26 24 pm" src="https://user-images.githubusercontent.com/96150256/217984971-db09cbf0-be89-4366-a232-d1515af8576b.png">